rotors_control/src/nodes/lee_position_controller_node.cpp

```diff
@@ -123,24 +107,22 @@ void LeePositionControllerNode::CommandMultiDofJointTrajectoryCallback(
     return;
   }
 
-  mav_msgs::EigenCommandTrajectoryPositionYaw eigen_reference;
-  mav_msgs::eigenCommandTrajectoryPositionYawFromMultiDofJointTrajectoryPointMsg(msg->points.front(),
-                                                                                 &eigen_reference);
+  mav_msgs::EigenTrajectoryPoint eigen_reference;
+  mav_msgs::eigenTrajectoryPointFromMsg(msg->points.front(), &eigen_reference);
```

Comment:
have a bool return type in the conversion function and check for its success here?